lib/tasks/manageiq/ui_tasks.rake

```diff
@@ -7,9 +7,14 @@ namespace :update do
           next
         end
         puts "== #{engine.name} =="
+        require 'json'
+        package_json = JSON.load_file("./package.json")
+        package_json["resolutions"] ||= {}
+        package_json["resolutions"]["es-abstract"] = "~1.21.2"
+        File.write("./package.json", JSON.pretty_generate(package_json))
```

Comment:
? Why is this needed?

Reply:
I was planning to update just this repo rather than locking down each plugin.  Unfortunately that fails tests.

Reply:
Ah ok